bayesian_cnn_prometheus/preprocessing/data_generator.py

```diff
@@ -0,0 +1,60 @@
+import functools
+from typing import Dict
+
+import numpy as np
+
+from bayesian_cnn_prometheus.constants import DatasetType
+from bayesian_cnn_prometheus.preprocessing.data_splitter import DataSplitter
+from bayesian_cnn_prometheus.preprocessing.preprocessing_links import ImageLoader, ImageNormalizer, ChunksGenerator
+
+
+class DataGenerator:
+    def __init__(self, preprocessing_config: Dict, batch_size: int):
+        """
+        Creates PreprocessingPipeline class to preprocess input data according to the config.
+        :param preprocessing_config: structure with configuration to use in preprocessing in learning
+        """
+        self.batch_size = batch_size
+        self.should_normalise = preprocessing_config.get("normalize_images").get("is_activated")
```

Comment:
Będzie błąd jak nie ma takiego klucza. 

I też z ciekawości, to czemu nie używasz `[]` to wyciągania wartości?

Reply:
Tak, sprawę z configiem zostawiłem dość luźno z wymienionego wyżej powodu :)

A w sumie nie wiedziałem, że `[]` jest tu obsługiwany.
Mam naleciałości z javy xD

Reply:
To jest poprawione ostatecznie?

Reply:
Nie pamiętam w tym momencie, ale config sam w sobie wartałoby zrefaktorować, dlatego mogłem to zostawić.
Trochę w myśl: nie ma sensu leczyć, coś co ma być usunięte